pandas/core/internals.py

```diff
@@ -95,6 +96,7 @@ class Block(PandasObject):
     is_object = False
     is_categorical = False
     is_sparse = False
+    is_extension = False
```

Comment:
you wouldn't do it like this. rather you would inherit from an ExtensionBlock

Reply:
I haven't looked at what these `is_` properties are used for, but all the other blocks had them so I added it for consistency.

Reply:
let's not add things like this until / unless necessary